### PR TITLE
[BPK-4390]: Migration component examples fixes

### PR DESCRIPTION
**Pull request by olliecurtis**
Includes fixes/improvements for examples found during docs site migration.

Also updates skip link to new format as there is an example on the docs site

### CODE REVIEW

**Review by shaundon — APPROVED** [review]

File: packages/bpk-component-skip-link/examples.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+import BpkSkipLink from './index';
+
+const DefaultExample = () => (
+  <BpkSkipLink
+    style={{ position: 'absolute' }}
+    href="#main"
+    label="Skip to main content"
+  />
+);
```
Comment (georgegillams) [inline packages/bpk-component-skip-link/examples.js:30]: We should have another example to jump to #implementation for the docs page

Comment (shaundon) [reply packages/bpk-component-skip-link/examples.js:30]: Ah good shout!

Comment (olliecurtis) [reply packages/bpk-component-skip-link/examples.js:30]: Ooo good shout!

File: packages/bpk-component-skip-link/examples.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+const DocsSiteExample = () => (
+  <BpkSkipLink
+    style={{ position: 'absolute' }}
+    href="#implementation"
+    label="Skip to implementation"
```
Comment (georgegillams) [inline packages/bpk-component-skip-link/examples.js:36]: My bad. It's called `readme`
```suggestion
    href="#readme"
    label="Skip to readme"
```

Comment (olliecurtis) [reply packages/bpk-component-skip-link/examples.js:36]: Not according to github 🤔 https://backpack.github.io/components/skip-link/?platform=web#implementation

Comment (olliecurtis) [reply packages/bpk-component-skip-link/examples.js:36]: This is also pulled from the live inspection of the code currently deployed

Screenshot 2021-03-17 at 13 27 24

**Pull request merged**